### PR TITLE
20992 - Create skeleton involuntary dissolutions job

### PR DESCRIPTION
**Pull request by chenhongjing**
*Issue #:* /bcgov/entity#20992

*Description of changes:*
- Create skeleton involuntary dissolutions job
- Add CI/CD workflow files for involuntary dissolutions job

*Things verified*
- Job runs at local docker

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the lear license (Apache 2.0).

### CODE REVIEW

File: jobs/involuntary-dissolutions/config.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    """Base class configuration."""
+
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    SENTRY_DSN = os.getenv('SENTRY_DSN', '')
+    LD_SDK_KEY = os.getenv('LD_SDK_KEY', None)
+
+    ACCOUNT_SVC_AUTH_URL = os.getenv('ACCOUNT_SVC_AUTH_URL', None)
+    ACCOUNT_SVC_CLIENT_ID = os.getenv('ACCOUNT_SVC_CLIENT_ID', None)
+    ACCOUNT_SVC_CLIENT_SECRET = os.getenv('ACCOUNT_SVC_CLIENT_SECRET', None)
```
Comment (argush3) [inline jobs/involuntary-dissolutions/config.py:54]: I don't think we'll need these for now.

File: jobs/involuntary-dissolutions/devops/vaults.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        ]
+    },
+    {
+        "vault": "entity",
+        "application": [
+            "postgres-legal",
+            "sentry",
+            "launchdarkly"
+        ]
+    }
```
Comment (argush3) [inline jobs/involuntary-dissolutions/devops/vaults.json:15]: Think we'll need to add `nats-filer`.

We'll need to probably publish a message to the filer topic so involuntary dissolution filings are processed.

**Review by tshyun24 — APPROVED** [review]
Doing my last approval😭

**Comment by argush3** [comment]
As per our conversation, please setup testing so we can add unit tests as we build out this job.

File: .github/workflows/involuntary-dissolutions-ci.yml [hunk 3]
```diff
@@ -47,22 +47,22 @@ jobs:
...
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          make setup
+      - name: Test with pytest
+        id: test
+        run: |
+          make test
```
Comment (argush3) [inline .github/workflows/involuntary-dissolutions-ci.yml:65]: Need to add environment variables for relevant config so tests will pass.

**Review by argush3 — APPROVED** [review]

**Review by leodube-aot — APPROVED** [review]

**Pull request merged**